diff --git a/org/apache/commons/configuration2/PropertiesConfiguration.java b/org/apache/commons/configuration2/PropertiesConfiguration.java
index 9ecc2be132816..373e23df17739 100644
--- a/org/apache/commons/configuration2/PropertiesConfiguration.java
+++ b/org/apache/commons/configuration2/PropertiesConfiguration.java
@@ -494,7 +494,7 @@ public class PropertiesConfiguration extends BaseConfiguration implements FileBa
                 return _emptyResult;
             }
 
-            return new String[] {line.substring(0, index).trim(), line.substring(index + 1), "="};
+            return new String[] {line.substring(0, index).trim(), line.substring(index + 1).trim(), "="};
         }
 
         private static final String[] _emptyResult = {"", "", ""};
